modules/ROOT/pages/whats-new-in-studio.adoc

```diff
@@ -1,31 +1,35 @@
 = What’s New in Studio
 
-Anypoint Studio 7.12 introduces support for comprehensive publication of assets to Exchange and for multi-factor authentication (MFA) to Anypoint Platform.
+Anypoint Studio 7.13 introduces support for message flow visualization and management of DataWeave libraries through its UI.
 
-This version also improves the metadata edition and Design Center import experiences and adds a feature that enables you to comment and uncomment objects directly from the canvas.
+This version also improves the general user experience including new features such as the Exchange Comprehensive Publication.
 
-== Comprehensive Publication to Exchange
+== Explore and Debug Shared Flows
 
-When publishing an asset to Exchange, you can now add more information (such as tags, categories, names, descriptions, and documentation) to your assets directly from Studio.
+Studio now supports message flow visualization of a Mule configuration loaded from a dependency to explore and debug shared flows.
```

Comment:
Mule configuration > Mule configuration file? (please confirm)
loaded from a dependency to explore... > loaded from a dependency, which enables exploring and debugging shared flows. (confirm if this is accurate)

Reply:
@Cristian-Venticinque yes Cristian both corrections are accurate.

Reply:
Thanks for the confirmation!